subsys/bluetooth/mesh/scheduler_srv.c

```diff
@@ -47,6 +54,25 @@ static bool set_second(struct tm *sched_time,
 		       struct bt_mesh_schedule_entry *entry,
 		       struct bt_mesh_time_srv *srv);
 
+static int store(struct bt_mesh_scheduler_srv *srv, uint8_t idx)
+{
+	char name[2] = {0};
+
+	if (!IS_ENABLED(CONFIG_BT_SETTINGS)) {
+		return 0;
+	}
+
+	name[0] = ENTRY_BASE_KEY + idx;
```

Comment:
I'm wondering why such complication when a simple operation is sufficient?

Reply:
It reads more naturally when logged (ie `bt-mesh/mod/v1234/data/5` instead of ending with /F). It's also convention throughout the mesh settings to encode it with proper numbers.

Reply:
Ok. We just had a mini-debate on conventions vs optimization somewhere below. This time I will follow convention then.